lib/Driver/Driver.cpp

```diff
@@ -2152,7 +2170,8 @@ void Driver::buildActions(SmallVectorImpl<const Action *> &TopLevelActions,
         (Triple.getObjectFormat() == llvm::Triple::ELF && !Triple.isPS4()) ||
         Triple.getObjectFormat() == llvm::Triple::Wasm ||
         Triple.isOSCygMing();
-    if (!AutolinkExtractInputs.empty() && AutolinkExtractRequired) {
+    if (!AutolinkExtractInputs.empty() && AutolinkExtractRequired &&
+        !shouldPerformLTO) {
```

Comment:
Does `shouldPerformLTO` influence the value of `RequiresAutolinkExtract`? Should it?